### DISCUSSION TITLE
[Feature] Allow sharing of an external library to more than one user

### DISCUSSION THREAD

**Discussion by kikendo** [immich-app/immich#14489] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I have two users in my Immich and I would like to have one of the libraries viewable by both users.
Creating a shared album is not what I want, I want to share the whole library.

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by zR-JB** [comment]
Why is this closed ?

**Comment by cholzer79** [comment]
@bo0tzz could you add a link to the other issue that this is a duplicate of when you close an issue?
Thanks!
I really need this feature too to share the existing library with all my family members.

**Reply by bo0tzz** [reply]
#4500
